pkg/namesgenerator/names-generator.go

```diff
@@ -15,68 +15,90 @@ var (
 		"amazing",
 		"angry",
 		"awesome",
-		"backstabbing",
-		"berserk",
-		"big",
```

Comment:
Combined with names they can be considered referring to body size, I thought that could possibly be offending.